package.json

```diff
@@ -167,6 +168,7 @@
     "event-stream": "3.3.2",
     "expect.js": "0.3.1",
     "faker": "1.1.0",
+    "fs-extra": "0.30.0",
```

Comment:
It looks like you're just using it for copying and deleting, we already have some utils you can use for that. `rimraf` for deleting, `ncp` for copying